File: SuperiorPackGroup/DAL/SPG.vb
Partial Class SPG
    Partial Public Class ShippingsDataTable
        Private Sub ShippingsDataTable_ColumnChanging(sender As Object, e As DataColumnChangeEventArgs) Handles Me.ColumnChanging
            If (e.Column.ColumnName = Me.strDeliveryNoteColumn.ColumnName) Then
                'Add user code here
            End If

        End Sub

    End Class
End Class

Namespace SPGTableAdapters
    
    Partial Class ShippingsTableAdapter

    End Class

    Partial Public Class ProductionTableAdapter
    End Class
End Namespace

Namespace SPGTableAdapters
    
    Partial Public Class ItemsTableAdapter
    End Class
End Namespace

Namespace SPGTableAdapters

    Partial Public Class ItemsTableAdapter
    End Class
End Namespace

File: SuperiorPackGroup/DAL/SPG.cs
//INSTANT C# NOTE: Formerly VB project-level imports:
using System;
using System.Collections;
using System.Collections.Generic;
using System.Data;
using System.Drawing;
using System.Diagnostics;
using System.Windows.Forms;

namespace SuperiorPackGroup
{
	public partial class SPG
	{
		public partial class ShippingsDataTable
		{
			private void ShippingsDataTable_ColumnChanging(object sender, DataColumnChangeEventArgs e)
			{
				if (e.Column.ColumnName == strDeliveryNoteColumn.ColumnName)
				{
					//Add user code here
				}

			}

		}
	}

	namespace SPGTableAdapters
	{

		public partial class ShippingsTableAdapter
		{

		}

		public partial class ProductionTableAdapter
		{
		}
	}

	namespace SPGTableAdapters
	{

		public partial class ItemsTableAdapter
		{
		}
	}

	namespace SPGTableAdapters
	{

		public partial class ItemsTableAdapter
		{
		}
	}

}